src/server/controllers/account-contributors.js

```diff
@@ -48,6 +48,25 @@ const contributorsQuery = gqlV2/* GraphQL */ `
                 createdAt
               }
             }
+            pausedRecurringContributions: orders(
+              searchTerm: $slug
```

Comment:
Here again, I think we should rather introduce an `account` parameter as `searchTerm` could return wrong values (eg. "opensource" could return an order with description "Contribution to Babel's OpenSource Fund").

Reply:
I'm looking at that.